Documentation/install/guides/kube-router.rst

```diff
@@ -0,0 +1,116 @@
+.. _kops_guide:
+
+****************************
+Using kube-router to run BGP
+****************************
+
+This guide explains how to configure Cilium and kube-router to co-operate to
+use kube-router for BGP peering and route propagation and Cilium for policy
+enforcement and load-balancing.
+
+.. note::
+
+    This feature is regarded as ``tech-preview``. Please provide feedback and
```

Comment:
shouldn't we use `alpha` or `beta`?

Reply:
I don't have a strong opinion as long as it is marked as pre stable. I've changed it to beta.